.editorconfig

```diff
@@ -0,0 +1,7 @@
+# http://editorconfig.org/
+
+root = true
+
+[*.{cc,h,py}]
```

Comment:
Should probably add `.c`, `.hh` to that list, along with the optional suffix `.in`
`.in` is used for templates of generated files
and `CMakeLists.txt`